greenplumpython/table.py

```diff
@@ -630,17 +633,19 @@ def _fetch(self, is_all: bool = True) -> Iterable[Tuple[Any]]:
         result = self._db.execute(to_json_table._build_full_query())
         return result if result is not None else []
 
-    def save_as(self, table_name: str, temp: bool = False, column_names: List[str] = []) -> "Table":
+    def save_into(
+        self, table_name: str, temp: bool = False, column_names: List[str] = []
+    ) -> "Table":
         """
-        Save the table to database as a real Greenplum Table
+        Save the **data** into a new :class:`Table`
```

Comment:
I actually don't want to hide the import details from users. This function DOES save (or "materialize" ) data into the database, and the information should be contained by the function description.

Do we still consider using `VTable` as the class name for GrenplumPython?

Some suggestion:

Save the **data** into the database, and return a new instance of  :class:`Table` which represents the newly saved data.

Reply:
I think we can consider `VTable`.

Here is the description of verticapy's `vDataFrame.to_db`: 

> Saves the vDataFrame current relation to the Vertica Database. This method can also be used to materialize flex tables.